test/test_samples_osw/base/results_output.csv

```diff
@@ -1,336 +1,336 @@
+OSW,door_area_ft_2,duct_unconditioned_surface_area_ft_2,electricity_bath_fan_kwh,electricity_ceiling_fan_kwh,electricity_clothes_dryer_kwh,electricity_clothes_washer_kwh,electricity_cooking_range_kwh,electricity_cooling_kwh,electricity_dishwasher_kwh,electricity_exterior_holiday_lighting_kwh,electricity_exterior_lighting_kwh,electricity_extra_refrigerator_kwh,electricity_fans_cooling_kwh,electricity_fans_heating_kwh,electricity_freezer_kwh,electricity_garage_lighting_kwh,electricity_heating_kwh,electricity_heating_supplemental_kwh,electricity_hot_tub_heater_kwh,electricity_hot_tub_pump_kwh,electricity_house_fan_kwh,electricity_interior_lighting_kwh,electricity_plug_loads_kwh,electricity_pool_heater_kwh,electricity_pool_pump_kwh,electricity_pumps_cooling_kwh,electricity_pumps_heating_kwh,electricity_pv_kwh,electricity_range_fan_kwh,electricity_recirc_pump_kwh,electricity_refrigerator_kwh,electricity_vehicle_kwh,electricity_water_systems_kwh,electricity_well_pump_kwh,floor_area_attic_ft_2,floor_area_conditioned_ft_2,floor_area_lighting_ft_2,fuel_oil_heating_mbtu,fuel_oil_water_systems_mbtu,hours_cooling_setpoint_not_met,hours_heating_setpoint_not_met,hvac_cooling_capacity_w,hvac_heating_capacity_w,hvac_heating_supp_capacity_w,include_enduse_subcategories,natural_gas_clothes_dryer_therm,natural_gas_cooking_range_therm,natural_gas_fireplace_therm,natural_gas_grill_therm,natural_gas_heating_therm,natural_gas_hot_tub_heater_therm,natural_gas_lighting_therm,natural_gas_pool_heater_therm,natural_gas_water_systems_therm,propane_clothes_dryer_mbtu,propane_cooking_range_mbtu,propane_heating_mbtu,propane_water_systems_mbtu,qoi_average_maximum_daily_timing_cooling_hour,qoi_average_maximum_daily_timing_heating_hour,qoi_average_maximum_daily_timing_overlap_hour,qoi_average_maximum_daily_use_cooling_kw,qoi_average_maximum_daily_use_heating_kw,qoi_average_maximum_daily_use_overlap_kw,qoi_average_minimum_daily_use_cooling_kw,qoi_average_minimum_daily_use_heating_kw,qoi_average_minimum_daily_use_overlap_kw,qoi_average_of_top_ten_highest_peaks_timing_cooling_hour,qoi_average_of_top_ten_highest_peaks_timing_heating_hour,qoi_average_of_top_ten_highest_peaks_use_cooling_kw,qoi_average_of_top_ten_highest_peaks_use_heating_kw,qoi_peak_magnitude_timing_hour,qoi_peak_magnitude_use_kw,roof_area_ft_2,size_cooling_system_kbtu_h,size_heating_system_kbtu_h,size_water_heater_gal,total_site_electricity_kwh,total_site_energy_mbtu,total_site_fuel_oil_mbtu,total_site_natural_gas_therm,total_site_propane_mbtu,total_site_wood_mbtu,wall_area_above_grade_conditioned_ft_2,wall_area_above_grade_exterior_ft_2,wall_area_below_grade_ft_2,window_area_ft_2,wood_heating_mbtu
```

Comment:
Why are we losing columns here?

Reply:
I may have accepted the old changes when merging develop, so the color index col was missing, re-running the CI now.